src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -278,6 +278,34 @@
     <Environment Issuer="https://api.epicgames.dev/epic/oauth/v1" />
   </Provider>
 
+  <!--
+                                      ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+                                      ██ ▄▄▄█ ▄▄▀██ ▄▄▀██ ▄▄▄██ ▄▄▀██ ▄▄▄ ██ ▄▄▄ ██ █▀▄██
+                                      ██ ▄▄██ ▀▀ ██ █████ ▄▄▄██ ▄▄▀██ ███ ██ ███ ██ ▄▀███
+                                      ██ ████ ██ ██ ▀▀▄██ ▀▀▀██ ▀▀ ██ ▀▀▀ ██ ▀▀▀ ██ ██ ██
+                                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
+  -->
+
+  <Provider Name="Facebook" Documentation="https://developers.facebook.com/docs/facebook-login/manually-build-a-login-flow">
+    <Environment Issuer="https://www.facebook.com/">
+      <Configuration AuthorizationEndpoint="https://www.facebook.com/v16.0/dialog/oauth"
+                     TokenEndpoint="https://graph.facebook.com/v16.0/oauth/access_token"
+                     UserinfoEndpoint="https://graph.facebook.com/v16.0/me">
+        <CodeChallengeMethod Value="S256" />
+      </Configuration>
+
+      <Scope Name="email" Default="true" Required="true" />
```

Comment:
Are we sure the `email` scope is really mandatory? Can you please try with this line removed?

Reply:
Yeah it works if you remove that line,  also works if you change it from Required to false.

Reply:
Great, thanks!